wagtail/admin/templates/wagtailadmin/home/workflow_pages_to_moderate.html

```diff
@@ -63,6 +65,9 @@ <h2 class="title-wrapper">{% trans 'Awaiting your review' %}</h2>
                                     </span>
                                 {% endfor %}
                             </td>
+                            <td valign="top">
+                                {{ revision.user }}
```

Comment:
suggestion:
```suggestion
                                {{ revision.user|user_display_name }}
```

for consistency with other areas where show the user.

Other than that LGTM

Reply:
discussed in Slack, will address during merge